netebpfext/user/fwp_um.cpp

```diff
@@ -68,6 +70,203 @@ _fwp_engine::classify_test_packet(_In_ const GUID* layer_guid, NET_IFINDEX if_in
     return result.actionType;
 }
 
+const uint32_t _test_destination_ipv4_address = 0x01020304;
```

Comment:
Generally speaking, IMO we should use the ```constexpr``` qualifier for compile time constants.

Reply:
Done